AutomationHelpers/test/WebLibraryTests.cs

```diff
@@ -116,6 +116,42 @@ public void WebLibraryTest_DownloadExeToSystemDir_Fail()
             //Assert
             Report.DetachLogger(logger);
             Assert.AreEqual(string.Format("Downloading a file from: {0} failed for the following reason:\r\nAn exception occurred during a WebClient request.\r\nAccess to the path '{1}' is denied.", address, fileName), logger.LastLogMessage);
+        }
+		
+		[Test]
+        public void WebLibraryTest_ResponseStatusCode_404()
+        {
+        	var logger = new TestReportLogger();

```

Comment:
Why do you need the logger here? This would just be needed to check for logged messages, but you don't check for them -> remove in all the 3 new unit tests?

Reply:
fixed